Updated importing-data-with-curl.md for Elasticsearch 8

epecchioli-florence-consulting opened the pull request:
How to properly use curl with Elasticsearch 8 is very well explained at the beginning of the course.
However I thought it would help everyone, to have the new syntax already available.

epecchioli-florence-consulting committed:
Update importing-data-with-curl.md

Updated the curl to work with Elasticsearch 8

ba0708 commented:
Thanks for the PR, @epecchioli-florence-consulting! Looks like I forgot to update that 🙂 

If you could make two small changes, I'll go ahead and merge this;
1. Omit the password since that's generally best practice (i.e. `-u elastic`)
2. Also show an example using `--cacert` ([like here](https://github.com/codingexplained/complete-guide-to-elasticsearch/blob/master/Getting%20Started/sending-queries-with-curl.md#2-provide-the-ca-certificate)) since that's technically best practice

Thanks!

epecchioli-florence-consulting committed:
Removed password in curl and added an example with --cacert

epecchioli-florence-consulting commented:
Hi Bo,
sure I can! I have updated the PR.
I added a generic --cacert [path_to_CA_certificate] instead of --cacert config/certs/http_ca.crt because we are in the bulk file directory (in the examples is Desktop) that could differ from the Elasticsearch installation folder.

I hope it's ok.

Thanks for your amazing course, I'm enjoying it a lot!

ba0708 commented:
Looks good, thanks @epecchioli-florence-consulting! And thank you for the kind words, I appreciate that! 🙂
